parsec/mca/termdet/fourcounter/termdet_fourcounter_module.c

```diff
@@ -0,0 +1,705 @@
+/**
+ * Copyright (c)      2018 The University of Tennessee and The University
```

Comment:
Incorrect copyright. In fact you should change all copyrights to be 2018-2022.

Reply:
All copyright of changed files should be updated now